cmd/podman/kube/play.go

```diff
@@ -341,14 +394,11 @@ func teardown(body io.Reader, options entities.PlayKubeDownOptions, quiet bool)
 	}
 
 	// Output stopped pods
-	if !quiet {
-		fmt.Println("Pods stopped:")
-	}
+	fmt.Println("Pods stopped:")
```

Comment:
This seems to revert the `!quiet` changes.  Did something go wrong when rebasing?

Reply:
Yeah, we removed the cleanup on error code as it was causing issues and `quiet` was introduced when I added the cleanup on error. With that gone, `quiet` was never set to true and go lint was unhappy with that. Since this is not being used anymore, I removed it.